mirdata/jams_utils.py

```diff
@@ -86,7 +87,12 @@ def jams_converter(
     # metadata
     if metadata is not None:
         for key in metadata:
-            if key == 'duration' and duration is not None and metadata[key] != duration:
+            if (
+                key == 'duration'
+                and duration is not None
+                and metadata[key] != duration
+                and 'spectrum' not in audio_path
```

Comment:
What is `spectrum` here?

Reply:
@genisplaja is this a mixup from cofla cante100 dataset? there is no spectrum in this dataset